.rhcicd/pr_check.sh

```diff
@@ -46,7 +46,7 @@ export APP_NAME="notifications"
 export COMPONENT_NAME="notifications-backend"
 export IMAGE="quay.io/cloudservices/notifications-backend"
 export DEPLOY_TIMEOUT="900"
-export EXTRA_DEPLOY_ARGS="--set-parameter notifications-engine/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-google-chat/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-microsoft-teams/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-servicenow/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-slack/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-splunk/IMAGE_TAG=${IMAGE_TAG} --no-remove-resources historical-system-profiles --no-remove-resources system-baseline"
+export EXTRA_DEPLOY_ARGS="--set-parameter notifications-engine/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-email/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-google-chat/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-microsoft-teams/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-servicenow/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-slack/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-splunk/IMAGE_TAG=${IMAGE_TAG} --no-remove-resources historical-system-profiles --no-remove-resources system-baseline"
```

Comment:
This will only work after we're done declaring `notifications-connector-email` in the deployment configuration in app-interface. I haven't checked if that's been merged already.

Reply:
No, it is failing because we are not pushing the images to the Quay repository yet. Do I leave this change out for another PR?

Reply:
Yes, we're not pushing, but bonfire should also complain after the image is pushed because notifications-connector-email isn't a known component (yet).

Reply:
From Jenkins:
```
05:40:54 2023-09-08 05:40:54 [   ERROR] [          MainThread] hit fatal error: component given for --set-parameter not found in app config: notifications-connector-email
```

Reply:
So this change needs to wait a bit. Let's remove it from this PR and merge it later when app-interface is ready.

Reply:
All right, removed!